services/m3dbnode/server/server_test.go

```diff
@@ -18,7 +18,7 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-// +build big
+// +build adhoc
```

Comment:
@richardartoul do we need to include these tags in the metalinter config too?

Reply:
I copied this out of Rob's pending PR and I think he wants to intentionally skip this test right now because its fails in Travis

Reply:
Oh I see what you're saying, we could include it, but I think this is a temporary hack we don't want to keep.

@robskillington

Reply:
Cool, I'll put a note in the other PR then.